app/assets/javascripts/jsxc.js

```diff
@@ -42,6 +42,11 @@ $(document).ready(function() {
             onlogin: true
           }
         });
+
+        // auto login if the user hasn't disabled it
+        if (!(app.currentUser.get("chat_auto_login"))) {
+          jsxc.gui.changePresence("online");
+        }
```

Comment:
Now you changed the name of the variable but kept the meaning, so if `chat_auto_login` is true it will disable the auto login. That's super confusing. I first thought you made a mistake here and only by reading the translations I was able to find out that this code is actually correct.

Please change the meaning of the variable and the translations so that we will log users in if `chat_auto_login` is enabled.